package.json

```diff
@@ -12,19 +12,20 @@
     "build": "rimraf dist && rollup -c rollup.config.js",
     "lint": "eslint --ext .js,.ts --ignore-path .gitignore --max-warnings 0 .",
     "lint:fix": "yarn lint --fix",
+    "tsc": "tsc",
```

Comment:
Consider adding `--noEmit` prevent TypeScript from making new files.

This command may be excessive because both Jest and Rollup check the types.